csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -213,6 +213,7 @@ private void TestPreTrainedModelsOpset7And8()
         {
             // 16-bit float not supported type in C#.
             var skipModels = new List<String>() {
+                "mxnet_arcface",
```

Comment:
This model has BN node with spatial = 0 (and with the new change - this model does not initialize - as is the expected behavior), so skipping this test

Reply:
Please add a comment saying why this model is being skipped as it has nothing to do with 16-bit float.

Reply:
Ok - I think the same is true for the skipped model - tf_inception_v2. @snnn mentioned that this model is failing on cpu but doesn't know the root cause. I will add a comment for this too.